### ISSUE TITLE
the dropout rate should be 0.2 in original paper?

### ISSUE THREAD

**Issue by linrongc** [tonylins/pytorch-mobilenet-v2#12]
https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet/mobilenet.py

**Comment by tonylins** [comment]
Hi,

Thanks. The original paper does not mention the dropout rate. But setting dropout rate to 0.5 is indeed my mistake, which makes the training much longer. I'll leave a notice in README.
